draft-thomassen-dnsop-generalized-dns-notify.md

```diff
@@ -389,9 +386,69 @@ records is ensured.
 NOTIFY(CSYNC) has the same timing consideration as NOTIFY(CDS) while
 NOTIFY(DNSKEY) does not.
 
+# The Format of the NOTIFY Record
+
+Here is the format of the NOTIFY RR, whose DNS type code is not yet
+defined.
+
+        Name TTL Class NOTIFY RRtype Scheme Port Target
+
+Name
+        The zone this RR refers to. In the case of NOTIFY(CDS) and
+        NOTIFY(CSYNC) this is the name of the parent zone. In the case
+        of NOTIFY(DNSKEY) this is the name of the zone in which a
+        change in the DNSKEY RRset is taking place.
+
+TTL
+        Standard DNS meaning [RFC 1035].
+
+Class
+        Standard DNS meaning [RFC 1035]. NOTIFY records occur in the IN
+        Class.
+
+RRtype
+        The type of generalised NOTIFY that this NOTIFY RR defines
+		the desired target address for. Currently only the types CDS,
+        CSYNC and DNSKEY are suggested to be supported, but there is
+		no protocol issue should a use case for additional types of
+		notifications arise in the future.
+ 		
+Scheme
+	    The scheme for locating the desired notification address.
+        The range is 0-255. This is a 16 bit unsigned integer in
+		network byte order. The value 0 is an error. The value 1 is
+		described in this document and all other values are currently
+		unspecified.
+
+Port
+	    The port on the target host of the notification service. The
+        range is 0-65535. This is a 16 bit unsigned integer in network
+		byte order.
+
+Target
+        The domain name of the target host providing the service of
+        listening for generalised notifications of the specified type.
+        There MUST be one or more address records for this name, the
+        name MUST NOT be an alias (in the sense of RFC 1034 or RFC
+        2181). 
+
+        A Target of "." means that the service is decidedly not
+        available at this domain.
```

Comment:
Is that different from not having the record at all?

Reply:
Hmm. While this is also stolen from RFC 2782 I did think about that. My thought was that it made sense to clearly signal the intention NOT to provide a target address for notifications, as a sort of "negative response" that clearly signals that there is no point in looking elsewhere. 

But on further reflection I think it is better to keep it simple so I removed this sentence also.